caffe2/CMakeLists.txt

```diff
@@ -1732,7 +1732,7 @@ if(BUILD_TEST)
   foreach(test_src ${Caffe2_CPU_TEST_SRCS})
     get_filename_component(test_name ${test_src} NAME_WE)
     add_executable(${test_name} "${test_src}")
-    target_link_libraries(${test_name} torch_library gtest_main)
+    target_link_libraries(${test_name} torch_library gtest_main stdc++)
```

Comment:
hmm, whats going on here? this is a pretty odd thing to have to add last minute, is it going to destabilize anything?

Reply:
missed the question: this was only for use at build time for tests / helps us use glog in the env for logging